portfolio/src/main/webapp/script.js

```diff
@@ -27,16 +27,23 @@ function addRandomFact() {
   factContainer.innerText = fact;
 }
 
-async function getMessage() {
-    // fetch('/data').then(response => response.json()).then((messages) => {
+function getMessage() {
```

Comment:
Function has get prefix, but did not return anything. Please add comment to explain what is intention of this function.

Reply:
It ends up getting a message out of the three possible messages. I added the code in a later commit. I think this commit was a mistake.